pygit2/__init__.py

```diff
@@ -59,6 +59,11 @@
 # libgit version tuple
 LIBGIT2_VER = (LIBGIT2_VER_MAJOR, LIBGIT2_VER_MINOR, LIBGIT2_VER_REVISION)
 
+# Let _pygit2 cache references to Python enum types.
+# This is separate from PyInit__pygit2() to avoid a circular import.
+cache_enums()
+del cache_enums  # Don't expose this to user code
```

Comment:
OK; in the latest commit, I had to import it explicitly from _pygit2 because `from _pygit2 import *` skips anything starting with an underscore.

Because the function now starts with an underscore, it won't be exposed to user code with `from pygit2 import *`, so I also removed the `del cache_enums` statement. (And if user code goes out of its way to call it anyway, the world won't stop spinning.)